### ISSUE TITLE
problem the result

### ISSUE THREAD

**Issue by MonsterCode2025** [CoderMusou/MECT4CNER#5]
dataset: msra
The rest is configured according to the readme section,
Why the result is not good?
^_^ please~
                                                                                                       
FitlogCallback evaluation on data-test:                                                                                                      
SpanFPreRecMetric: f=0.536364, pre=0.553125, rec=0.520588                                                                                    
label_acc: acc=0.957234
Evaluation on dev at Epoch 27/100. Step:3429/12700:                                                                                          
SpanFPreRecMetric: f=0.536364, pre=0.553125, rec=0.520588                                                                                    
label_acc: acc=0.957234

**Comment by CoderMusou** [comment]
Did you finish 100 epochs?

**Comment by MonsterCode2025** [comment]
> Did you finish 100 epochs?

Yes, I finished 100 epochs. But the result of p ,r and f1 are more than 70%

**Comment by kbprogrammer-tech** [comment]
Can you tell me how to get the structural components (radicals) of Chinese characters? Can you share them

**Comment by MonsterCode2025** [comment]
> Can you tell me how to get the structural components (radicals) of Chinese characters? Can you share them

The link is in readme, https://github.com/kfcd/chaizi

**Comment by CoderMusou** [comment]
> Yes, I finished 100 epochs. But the result of p ,r and f1 are more than 70%

Where is your dataset collected?

**Comment by MonsterCode2025** [comment]
The data set I collected came from the original paper, and the source of the data set was mentioned in the original paper


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "CoderMusou/MECT4CNER"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月10日(星期五) 下午2:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [CoderMusou/MECT4CNER] problem the result (Issue #5)





  
Yes, I finished 100 epochs. But the result of p ,r and f1 are more than 70%
 Where is your dataset collected?
  
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by playermyx** [comment]
你好，我复现实验运行“python main.py --dataset weibo”这一步时报错IndexError: list index out of range（MECT4CNER\Modules\CNNRadicalLevelEmbedding.py第26行），请问这是什么原因呢，需要做哪写修改呢
